x-pack/test/functional_with_es_ssl/apps/discover_ml_uptime/ml/alert_flyout.ts

```diff
@@ -114,6 +114,14 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
         await ml.alerting.assertTopNBuckets(1);
         await ml.alerting.assertLookbackInterval('123m');
 
+        await ml.testExecution.logTestStep(
+          'should support updating the advanced settings section with valid values only'
+        );
+        await ml.alerting.setTopNBuckets(0, true);
+        await ml.alerting.setTopNBuckets(2);
```

Comment:
Nit - could add an extra check for a non integer value too.

Reply:
I had to revert it, because it's quite tricky to test. Essentially the browser doesn't let you setting a non integer values. An attempt to set a number with floating point, e.g. `2.5` results in `05` input value.